src/components/implementation/cbuf_mgr/default/cbuf_mgr.c

```diff
@@ -9,16 +9,27 @@
 
 #include <cos_component.h>
 #include <mem_mgr_large.h>
-#include <cbuf.h>
+#include <cbuf_meta.h>
 #include <cbuf_mgr.h>
 #include <cos_synchronization.h>
 #include <valloc.h>
+#include <sched.h>
 #include <cos_alloc.h>
 #include <cmap.h>
 #include <cos_list.h>
 
 #include <stkmgr.h>
 
+/* debug helper */
+#define OP_NUM 10
+/* 0-create 1-collect 2-delete */
+/* 3-retrieve 4-register */
+/* 56-reserve 7-cbuf_alloc_map */
+int op_nums[OP_NUM];
+unsigned long long per_total[OP_NUM];
+
+#define INIT_TARGET_SIZE 4096*2048
```

Comment:
this is the default size of cbuf pool for each components.
There is no hard reason for this value, it just cannot be too small.